saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/metadata/OpenSamlMetadataResolver.java

```diff
@@ -121,6 +123,10 @@ public void setEntityDescriptorCustomizer(Consumer<EntityDescriptorParameters> e
 		this.entityDescriptorCustomizer = entityDescriptorCustomizer;
 	}
 
```

Comment:
Will you please add a brief JavaDoc? Something like this:

```java
/**
  * Configure whether to pretty-print the metadata XML. This can be
  * helpful when signing the metadata payload.
  * @since 6.2
  **/
```

Reply:
yes, done, i added your suggestion, thank you!